components/Button/styles.css

```diff
@@ -61,7 +61,7 @@
 	position: fixed;
 }
 .btn-floating:hover {
-	background-color: var(--theme-primary-light);
+	background-color: var(--theme-primary-400);
 }
 .btm-r {
```

Comment:
ig class name should be btn-r

Reply:
Actually it's correct  btm-r =>bottom right